### DISCUSSION TITLE
If I want to flash the Ubuntu system onto a microcontroller board and run this software, what are the performance requirements, and are there any recommended compatible hardware in terms of driver support?

### DISCUSSION THREAD

**Discussion by longwenzhang1991** [flowdriveai/flowpilot#39] [Q&A]
If you want to flash the Ubuntu system onto a microcontroller board and run this software, what are the performance requirements, and are there any recommended compatible hardware in terms of driver support?

**Comment by RaghavPrabhakar66** [comment]
Depends on the SBC. We have tried on Jetson nano and Raspberry pi4. The performance was unsatisfactory.

**Comment by MankaranSingh** [comment]
get opencl working and that should be pretty much it.
